src/CTA.Rules.Actions/ActionHelpers/ConfigMigrate.cs

```diff
@@ -22,6 +22,15 @@ public ConfigMigrate(string projectDir, ProjectType projectType)
             _projectType = projectType;
             _hasData = false;
         }
+
+        public void WebformsWebConfigMigrateHelper()
+        {
+            Configuration configXml = LoadWebConfig(_projectDir);
+            var templateContent = @"{}";
+            JObject config = ProcessWebConfig(configXml, templateContent);
+            AddAppSettingsJsonFile(config, _projectDir);
```

Comment:
If you make this return `config.ToString()` we can just use that content directly instead of calling `AddAppSettingsJsonFile` and having to re-read then write that content

Reply:
Well you don't have to do the byte conversion in there but you can if you want. I just meant that we should directly return the conversion result instead of writing it to disk then re-reading and re-writing